src/Microsoft.ML.MamlExec/Microsoft.ML.MamlExec.csproj

```diff
@@ -0,0 +1,28 @@
+﻿<Project Sdk="Microsoft.NET.Sdk">
+
+  <PropertyGroup>
+    <AllowUnsafeBlocks>true</AllowUnsafeBlocks>
+    <DefineConstants>CORECLR</DefineConstants>
+    <TargetFramework>netcoreapp2.0</TargetFramework>
+	<IncludeInPackage>Microsoft.ML</IncludeInPackage>
+	<OutputType>Exe</OutputType>
+	<StartupObject>Microsoft.ML.Runtime.Tools.Maml</StartupObject>
+  </PropertyGroup>
+
+  <ItemGroup>
+    <ProjectReference Include="..\Microsoft.ML.Core\Microsoft.ML.Core.csproj" />
+	<Compile Include="..\Microsoft.ML.Maml\MAML.cs">
+      <Link>MAML.cs</Link>
+    </Compile>
+	<Compile Include="..\Microsoft.ML.Maml\ChainCommand.cs">
+      <Link>ChainCommand.cs</Link>
+    </Compile>
+	<Compile Include="..\Microsoft.ML.Maml\HelpCommand.cs">
+      <Link>HelpCommand.cs</Link>
+    </Compile>
+	<Compile Include="..\Microsoft.ML.Maml\VersionCommand.cs">
+      <Link>VersionCommand.cs</Link>
+    </Compile>
```

Comment:
This arrangement is very peculiar. Now there are two projects in the solution with the same files. Let's do something else?

Reply:
I'm going to address this but will leave it active until then.

---
In reply to: [191493310](https://github.com/dotnet/machinelearning/pull/248#discussion_r191493310) [](ancestors = 191493310)